tests/test_mark_tokens.py

```diff
@@ -550,3 +561,21 @@ def test_complex_slice_and_parens(self):
     source = 'f((x)[:, 0])'
     m = self.create_mark_checker(source)
     m.verify_all_nodes(self)
+
+  def test_sys_modules(self):
+    for module in list(sys.modules.values()):
+      try:
+        filename = inspect.getsourcefile(module)
+      except TypeError:
+        continue
+
+      if not filename:
+        continue
+
+      filename = os.path.abspath(filename)
+      print(filename)
+      with io.open(filename) as f:
+        source = f.read()
+      source = source.replace('elif', 'if')
```

Comment:
what's this about?

Reply:
Sorry, I should probably add a comment. `elif:` is really just `else: if:` to the AST, so `get_text` can return text starting with `elif` when given an `If` node. I think this is generally harmless and there's probably no good alternative, but it leads to syntax errors in `verify_all_nodes`.

Reply:
How about, in `verify_all_nodes`, if the text starts with `/^\s*elif\s`, replace "elif" with "if"? (With your comment to explain.) It would be clearer that this is specific to how nodes are verified, and would apply in all cases, not just in one test case.